### PR TITLE
[spb] Update russian translation of "Installation" and "What's next?"

### PR DESCRIPTION
**Pull request by istanya**
Changes in this pull request:

"Installation" section:

* added section "If you're attending a workshop"
* added section "Brief intro to the command line"
* minor fixes

"What's next?" section:

* added links to resources
* minor fixes

**NB**: there will be several pull-requests to russian translation and we will review each others changes.

### CODE REVIEW

File: ru/installation/README.md [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
 
-Дело в том, что мы всё равно рассматриваем все эти вещи по ходу руководства, а в этом разделе мы просто собрали все инструкции по установке вместе. Встречи Django Grils включают в себя "вечер настроек", когда мы устанавливаем всё что нужно, чтобы не тратить на это время в течение основного семинара, это очень удобно для нас.
-
-Если ты также считаешь эту идею удачной, то можешь следовать содержанию главы. Чтобы сразу перейти к изучению нового материала, пропустив установку необходимых программ, просто пропусти эту главу - мы будем объяснять все необходимые вещи по мере продвижения по руководству.
+Дело в том, что по ходу руководства мы всё равно будем рассматривать все эти вещи, а текущий раздел является дополнительным, где мы просто собрали инструкции по установке вместе (это используется в некоторых форматах мастер-классов). Ты можешь установить все сразу, воспользовавшись инструкцией ниже. Но можешь пропустить этот раздел, приступив к изучению основного материала, где детали установки мы объясним по мере необходимости.
 
 Удачи!
 
-# Установка
+# Если ты посещаешь семинар
```
Comment (b0g3r) [inline ru/installation/README.md]: Не уверен, нужно будет подумать про формулировку. Может мастер-класс, как на https://djangogirls.org/stpetersburg/
```suggestion
# Если ты посещаешь воркшоп
```

File: ru/whats_next/README.md [hunk 2]
```diff
@@ -6,35 +6,33 @@
...
- [4]: https://docs.djangoproject.com/en/1.11/intro/tutorial01/
- [5]: http://newcoder.io/tutorials/
- [6]: https://www.codecademy.com/en/tracks/python
- [7]: https://www.codecademy.com/tracks/web
- [8]: https://github.com/ggcarrots/django-carrots/
- [9]: http://learnpythonthehardway.org/book/
- [10]: http://www.gettingstartedwithdjango.com/
- [11]: https://www.twoscoopspress.com/products/two-scoops-of-django-1-11
- [12]: https://hellowebapp.com/
+Конечно! Сейчас доступно _огромное_ количество ресурсов для изучения всевозможных навыков программирования. Мы поможем подобрать тебе то, что будет полезно изучить на следующем этапе. Вот несколько бесплатных ресурсов(или ресурсов с большим количеством бесплатного контента):
```
Comment (b0g3r) [inline ru/whats_next/README.md]:
```suggestion
Конечно! Сейчас доступно _огромное_ количество ресурсов для изучения всевозможных навыков программирования. Мы поможем подобрать тебе то, что будет полезно изучить на следующем этапе. Вот несколько бесплатных ресурсов (или ресурсов с большим количеством бесплатного контента):
```

File: ru/installation/README.md (hunk 1 above)
Comment (LenaVolzhina) [reply ru/installation/README.md]: Много где было использовано слово "руководство"

File: ru/installation/README.md [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
+# Установка Python
+{% include "/python_installation/instructions.md" %}
 
 # Установка редактора кода
-
 {% include "/code_editor/instructions.md" %}
 
-# Установка Git
+# Настрока virtualenv и установка Django
+{% include "/django_installation/instructions.md" %}}
```
Comment (b0g3r) [inline ru/installation/README.md]: Нашёл из-за чего проваливается сборка travis
```suggestion
{% include "/django_installation/instructions.md" %}
```

File: ru/installation/README.md [hunk 4]
```diff
@@ -1,39 +1,46 @@
...
+- Если мастер-класс, участницей которого ты являешься, не организует встречу "вечер настроек" (ты не смогла присутствовать), и организаторы не попросили тебя попытаться установить все до начала мероприятия, пропусти этот раздел и переходи к главе [Как работает интернет](../how_the_internet_works/README.md). Не переживай, ты установишь все необходимое во время прохождения руководства на самом мастер-классе.
 
-# Установка Python
+# Установка
+На мастер-классе ты будешь создавать блог. Также по мере необходимости, ты узнаешь, как устанавливать различное программное обеспечение и создашь некоторые учетные записи. Данная страница является местом, где собраны вместе все необходимые инструкции по установке и регистрации.
 
-{% include "/python_installation/instructions.md" %}
+<!--sec data-title="Настройка Chromebook (если ты используешь его)"
+data-id="chromebook_setup" data-collapse=true ces-->
```
Comment (b0g3r) [inline ru/installation/README.md]: Ещё из возможных проблем: перенос строки здесь лишний и ломает тег

File: ru/installation/README.md [hunk 5]
```diff
@@ -1,39 +1,46 @@
...
+{% include "/chromebook_setup/instructions.md" %}
```
Comment (b0g3r) [inline ru/installation/README.md]: И он не сможет подтянуть этот файл, пока пулл реквест #1484 не будет смерджен https://github.com/DjangoGirls/tutorial/pull/1484/files#diff-9217c652f280eda91d39eacd9b2ddd3b

Comment (b0g3r) [reply ru/installation/README.md]: Так что ждёмс 🌝

Comment (OPersian) [reply ru/installation/README.md]: @istanya thanks a lot for your contribution!
Please remove the `sec` and `endsec` lines, as also there are no such lines in the [original file](https://github.com/DjangoGirls/tutorial/blob/e3de2cc0d356ab0922da21110440408cc2aa748d/en/installation/README.md#installation).
Thanks in advance!

File: ru/installation/README.md (hunk 4 above)
Comment (istanya) [reply ru/installation/README.md]: Скорее всего это связано с тем, что файла ru/chromebook_setup/instructions.md в главном репозитории пока нет. Перенос тега в англоязычной версии нормально воспринимается.
Спасибо тебе большое за ревью!))

File: ru/installation/README.md (hunk 5 above)
Comment (istanya) [reply ru/installation/README.md]: В режиме редактирования (raw) они присутствуют

Comment (istanya) [reply ru/installation/README.md]: В режиме редактирования (raw) `sec` and `endsec` присутствуют
Думаю сейчас Travis построит успешно, т.к. смержен https://github.com/DjangoGirls/tutorial/pull/1484 и файл  ru/chromebook_setup/instructions.md в главном репозитории теперь присутствует

**Comment by OPersian** [comment]
Ah sorry, my bad)

**Comment by magul** [comment]
There's [Travis build](https://travis-ci.org/DjangoGirls/tutorial/builds/494829443) failing here. You should fix that before it would be merged.

**Comment by LenaVolzhina** [comment]
@magul as far as I see, Travis build failed because Chromebook setup instructions were absent in master branch at the moment this PR was made. But now they are already merged (https://github.com/DjangoGirls/tutorial/pull/1484/files#diff-9217c652f280eda91d39eacd9b2ddd3b), so it should be OK now. Could you please restart Travis build?

**Comment by magul** [comment]
@LenaVolzhina I just re-run it but I doubt that it will change base commit of the pull request. I believe that @istanya  needs to merge `master` back to that pull request's branch to resolve that issue.

**Comment by magul** [comment]
I looks that I was pesimistic here. Merging! :tada:

**Comment by LenaVolzhina** [comment]
@magul Haha, seems like Travis is more intelligent than we thought. Thanks a lot for your help.

@OPersian thank you too for quick but thoughtful review of all our pull-requests!

**Pull request merged**